### PR TITLE
Remove dependency on FlutterAppDelegate so devs can more easily add SDKs.

### PR DESCRIPTION
**Pull request by collinjackson**
FlutterAppDelegate will be removed from engine repo in flutter/engine#2792

### CODE REVIEW

**Comment by collinjackson** [comment]
@Hixie

File: packages/flutter_tools/templates/create/ios.tmpl/Runner/AppDelegate.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#import <UIKit/UIKit.h>
+
+@interface AppDelegate : UIResponder <UIApplicationDelegate>
+
+@property (strong, nonatomic) UIWindow *window;
+
+
```
Comment (abarth) [inline packages/flutter_tools/templates/create/ios.tmpl/Runner/AppDelegate.h]: Extra blank line here.

File: packages/flutter_tools/templates/create/ios.tmpl/Runner/AppDelegate.m [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#import <Flutter/Flutter.h>
+#include "AppDelegate.h"
+
+@implementation AppDelegate
+
+
```
Comment (abarth) [inline packages/flutter_tools/templates/create/ios.tmpl/Runner/AppDelegate.m]: extra blank line here.

**Comment by collinjackson** [comment]
(FYI, the blank lines were put there intentionally to match the default AppDelegate files that Xcode gives you, but I'll change them to match Flutter style.)

**Pull request merged**